packages/nodes-base/nodes/Airtable/v2/methods/loadOptions.ts

```diff
@@ -25,13 +25,22 @@ export async function getColumns(this: ILoadOptionsFunctions): Promise<INodeProp
 		});
 	}
 
+
+	const useFieldIDs = this.getNodeParameter('useFieldIDs', undefined, false) as boolean;
+
 	const result: INodePropertyOptions[] = [];
 
 	for (const field of tableData.fields as IDataObject[]) {
+
+		const fieldLabel = useFieldIDs ? field.id : field.name; // Use field ID if flag is set
+		const fieldDescription = `Type: ${field.type}`;
+
+
+
 		result.push({
-			name: field.name as string,
-			value: field.name as string,
-			description: `Type: ${field.type}`,
+			name: fieldLabel as string,
```

Comment:
why are we replacing name with id if `useFieldIDs` is true? I am concerned the naming will become confusing. and won't we always need the `name` for the UI, even if `useFieldIDs` is true?

```suggestion
			name: field.name as string,
			id: field.id as string,
```

Reply:
Resolved in commit 

commit 6093ab98d1ecccf3d901625c0a4b5bfa6733c19c 

Description 

Updated the retrieval of baseId and tableId by using undefined as the fallback value, ensuring clarity in cases where these parameters are missing. If either baseId or tableId is not provided, a NodeOperationError is thrown to prevent further execution with incomplete data. This enhances error handling by explicitly requiring both parameters for correct function operation.